test/regressions/index.js

```diff
@@ -27,7 +33,22 @@ importRegressionFixtures.keys().forEach((path) => {
 }, []);
 
 const blacklist = [
-  'docs-getting-started-templates-dashboard-internals-components/CustomIcons.png', // No public components
+  // Next components are tested by docs-getting-started-templates-dashboard-components/MainGrid.png
```

Comment:
```suggestion
  // The following components are tested by docs-getting-started-templates-dashboard-components/MainGrid.png
```

The downside is that this list must be manually synchronized with the demos. But it's still better than running all demos unnecessarily, I guess.